address-space-controller/src/main/java/io/enmasse/controller/PodDisruptionBudgetController.java

```diff
@@ -159,4 +159,9 @@ private void reconcileBrokerPodDisruptionBudget(AddressSpace addressSpace, Stand
             deleteIfExists(name);
         }
     }
+
+    @Override
+    public String toString() {
+        return "PodDisruptionBudgetController";
```

Comment:
Maybe add a `namespace` value here for more details

Reply:
The namespace should be the same as where infra is deployed, so I'd like to follow the convention for the other controllers added to the chain, otherwise the log will look strange.